pyproject.toml

```diff
@@ -51,19 +45,18 @@ importlib-metadata = {version = "^1.6.0", python = "<3.8"}
 inflection = "^0.5.1"
 jsonschema = "^3.2.0"
 networkx = "^2.5"
+oauth2client = "<4.0.0"  # Specified because of bug in version ^4.0.0
```

Comment:
yes, we need to evaluate our codebase and specify what would need to be done to to transition to later versions of this lib; but that's different PR :)